src/services/identifierService.ts

```diff
@@ -17,24 +17,9 @@ export interface IdentifierInterface {
   create(userId: string, req: IdentifierCreateReq): Promise<IdentifierRsp>;
   delete(userId: string, identifierId: string): Promise<GenericRsp>;
   list(sort?: string, filter?: string[], page?: number, pageSize?: number): Promise<IdentifierList>;
-  listAllWithPaging(page?: number, pageSize?: number): Promise<IdentifierList>;
   listByValueAndType(value: string, type: IdentifierType): Promise<IdentifierList>;
```

Comment:
We have 4 list methods now which is good. As mentioned in my previous comment all of them should support paging (and thus also sorting). So please check the comment again.

As an order for the parameters I suggest:
- filter(s)
- sorting
- page
- pagesize

e.g. 
list(filter?: string[], sort?: string, page?: number, pageSize?: number): Promise<IdentifierList>;
listByUserId(userId: string, sort?: string, page?: number, pageSize?: number): Promise<IdentifierList>;